frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -132,46 +130,69 @@ export const startPipeline = (
       triggerPipeline(pipeline, onSubmit);
     }
   },
+  accessReview: {
+    group: kind.apiGroup,
+    resource: kind.plural,
+    name: pipeline.metadata.name,
+    namespace: pipeline.metadata.namespace,
+    verb: 'create',
+  },
 });
-export const rerunPipeline = (
+
+export const rerunPipeline: KebabAction = (
+  kind: K8sKind,
   pipeline: Pipeline,
   latestRun: PipelineRun,
   onSubmit?: (pipelineRun: PipelineRun) => void,
-): ActionFunction => {
+) => {
   if (!latestRun || !latestRun.metadata) {
     // The returned function will be called using the 'kind' and 'obj' in Kebab Actions
-    return (): Action => ({
-      label: <div className="dropdown__disabled">Start Last Run</div>,
-      callback: null,
-    });
+    return {
+      label: 'Start Last Run',
+      hidden: true,
```

Comment:
Why is this always hidden?
I thought hidden would omit the item from the menu. So if it's always hidden, why return any value at all?
Previously the menu item was disabled.

Reply:
Fixed.